CHANGELOG.next.asciidoc

```diff
@@ -44,6 +44,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Update Salesforce module to use new Salesforce input. {pull}37509[37509]
 - Tag events that come from a filestream in "take over" mode. {pull}39828[39828]
 - Fix high IO and handling of a corrupted registry log file. {pull}35893[35893]
+- Enable StatusReporter for log input. {pull}40075[40075]
```

Comment:
Same thing as the Filestream one, we should write changelog from the perspective of an user that does not know the internals of Beats.

Reply:
Should be something like:
```suggestion
- Enable file ingestion to report detailed status to Elastic Agent {pull}40075[40075]
```